packages/synapse-react-client/src/components/dataaccess/UserAccessRequestHistory/UserAccessRequestHistoryPlace.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  BreadcrumbItem,
+  Breadcrumbs,
+} from '@/components/Breadcrumbs/Breadcrumbs'
+import { Box } from '@mui/material'
+import { useMemo } from 'react'
+import {
+  createBrowserRouter,
+  createMemoryRouter,
+  Navigate,
+  Outlet,
+  RouteObject,
+  RouterProvider,
+  useLocation,
+  useParams,
+} from 'react-router'
+import { RouterProvider as DOMRouterProvider } from 'react-router/dom'
+import SubmissionPage from '../SubmissionPage/SubmissionPage'
+import {
+  SUBMISSION_ID_PARAM,
+  USER_ACCESS_HISTORY_SUBMISSION_SUBPATH,
+} from './RouteConstants'
+import { UserAccessRequestHistoryPage } from './UserAccessRequestHistoryPage'
+
+type ReviewerDashboardProps = {
+  /** Used to determine the base path for the component. Default is RequestHistory:default */
+  routerBaseName?: string
+  /** If true use a MemoryRouter, which prevents the browser URL from updating. For demo/test purposes only. */
+  useMemoryRouter?: boolean
+}
+
+function UserSubmissionPageRouteRenderer() {
+  const { submissionId } = useParams<{ [SUBMISSION_ID_PARAM]: string }>()
+  if (!submissionId) {
+    return <Navigate to="/" />
```

Comment:
If a submission id was unavailable on the Submission page (which seems like a very rare edge case), I would think it would ideally navigate to the reviewer dashboard router base name (/RequestHistory:default by default).   If this is indeed very rare, this code probably does not need to be changed

Reply:
The `to` is within the context of the baseName, so this will already work how you expect in SWC.